adal/src/main/java/com/microsoft/aad/adal/ADALError.java

```diff
@@ -554,7 +554,7 @@
     
     private String mDescription;
 
-    private ADALError(String message) {
+    ADALError(String message) {
```

Comment:
I don't see a down side of doing this at this point, so i would like to leave it as is.